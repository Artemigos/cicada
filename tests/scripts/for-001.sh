echo hi
for var in foo bar baz
    echo $var
done

for var2 in $(echo a b)
    echo hello && echo $var2
done

for var3 in 'args kwargs' "sh script"
    echo $var3
done

for var4 in src/builtins/ex*.rs
    echo source file $var4
done

for c in ab , ' ' xy
    echo "append '$c' into file"
done

echo bye
