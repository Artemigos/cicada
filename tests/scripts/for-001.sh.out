hi
foo
bar
baz
hello
a
hello
b
args kwargs
sh script
source file src/builtins/exec.rs
source file src/builtins/exit.rs
source file src/builtins/export.rs
append 'ab' into file
append ',' into file
append ' ' into file
append 'xy' into file
bye
